clients/html/src/app/employer-details/employer-details.component.html

```diff
@@ -137,7 +137,7 @@ <h2 class="heading-text">Employee Roster</h2>
             Note: You may use this complete roster download when you apply.
           </p>
           <button type="button" class="btn btn-link">
-            <a href="/get-a-quote/assets/roster_upload_template.xlsx" download
+            <a href="/assets/roster_upload_template.xlsx" download
```

Comment:
Did you tested download locally with this change?

Reply:
Yes, I did a test in my local, and it's downloading.